Source/Diagnostics/ComputeDiagFunctors/DivEFunctor.cpp

```diff
@@ -3,18 +3,21 @@
 
 using namespace amrex;
 
-DivEFunctor::DivEFunctor(const std::array<const amrex::MultiFab* const, 3> arr_mf_src, const int lev, const int ncomp)
-    : ComputeDiagFunctor(ncomp), m_arr_mf_src(arr_mf_src), m_lev(lev)
+DivEFunctor::DivEFunctor(const std::array<const amrex::MultiFab* const, 3> arr_mf_src, const int lev, const amrex::IntVect crse_ratio, const int ncomp)
+    : ComputeDiagFunctor(ncomp, crse_ratio), m_arr_mf_src(arr_mf_src), m_lev(lev)
 {}
 
 void
 DivEFunctor::operator()(amrex::MultiFab& mf_dst, const int dcomp) const
 {
     auto& warpx = WarpX::GetInstance();
-
+    const int ng = 1;
```

Comment:
```suggestion
    constexpr int ng = 1;
```
actually do we really need a guard cell here?

Reply:
Yes. we do. Here we are computing divE and passing it as a source MF to AverageAndCoarsen. When interpolating data from the src_mf (that stored divE) to the output diagnostic mf_avg, we need guard cells from src_mf.